src/corredor/helpers/compose.ts

```diff
@@ -159,7 +242,7 @@ export class Compose {
    * @returns {String}
    * @private
    */
-  makeRecordPageTitle ({ name }) {
+  makeRecordPageTitle ({ name }: { name: string}) {
```

Comment:
I've talked w/ @tjerman about this.
this function is a bit too specific to be in helpers. 

Remove it please.